### ISSUE TITLE
Build windows app on macos 

### ISSUE THREAD

**Issue by lexoveri** [microsoft/react-native-windows#5925]
How can I build Windows application  on my macbook pro?

**Comment by asklar** [comment]
you should be able to create a Windows virtual machine and install the necessary tools via the rnw-dependencies.ps1 script. See the docs at https://microsoft.github.io/react-native-windows/docs/rnw-dependencies

**Comment by chrisglein** [comment]
Yep, that's pretty much it - get a VM and then follow our steps.
This is a common question and is worth us having some documentation on. I'm going to migrate this issue to our website repo where we track documentation asks.
